.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
```

Comment:
Can we keep these please? it converts `foo.default` to `foo['default']` so things don't break in safari 5